<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-EGH5GRK4" ignored="0" name="注册接口" tests="6" failures="6" timestamp="23 八月 2020 04:14:18 GMT" time="5.117" errors="0">
  <testcase name="test" time="1.785" classname="com.api.cases.RegisterCase">
    <failure type="java.lang.AssertionError" message="expected [passed] but found [failed]">
      <![CDATA[java.lang.AssertionError: expected [passed] but found [failed]
at com.api.cases.RegisterCase.test(RegisterCase.java:47)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.096" classname="com.api.cases.RegisterCase">
    <failure type="java.lang.AssertionError" message="expected [passed] but found [failed]">
      <![CDATA[java.lang.AssertionError: expected [passed] but found [failed]
at com.api.cases.RegisterCase.test(RegisterCase.java:47)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.093" classname="com.api.cases.RegisterCase">
    <failure type="java.lang.AssertionError" message="expected [passed] but found [failed]">
      <![CDATA[java.lang.AssertionError: expected [passed] but found [failed]
at com.api.cases.RegisterCase.test(RegisterCase.java:47)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.093" classname="com.api.cases.RegisterCase">
    <failure type="java.lang.AssertionError" message="expected [passed] but found [failed]">
      <![CDATA[java.lang.AssertionError: expected [passed] but found [failed]
at com.api.cases.RegisterCase.test(RegisterCase.java:47)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="1.885" classname="com.api.cases.RegisterCase">
    <failure type="java.lang.AssertionError" message="expected [passed] but found [failed]">
      <![CDATA[java.lang.AssertionError: expected [passed] but found [failed]
at com.api.cases.RegisterCase.test(RegisterCase.java:47)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.162" classname="com.api.cases.RegisterCase">
    <failure type="java.lang.AssertionError" message="expected [passed] but found [failed]">
      <![CDATA[java.lang.AssertionError: expected [passed] but found [failed]
at com.api.cases.RegisterCase.test(RegisterCase.java:47)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- 注册接口 -->
